.ci/create-changes-html.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+if [ $# != 2 ]; then
+    echo >&2 "usage: $0 BASE_DOC_COMMIT DOC_REPOSITORY"
+    echo >&2 "creates CHANGES.html in the current directory"
+    echo >&2 "for the diffs of DOC_REPOSITORY against BASE_DOC_COMMIT"
+    exit 1
+fi
+BASE_DOC_COMMIT="$1"
+DOC_REPOSITORY="$2"
+
+# Wipe out chronic diffs between old doc and new doc
+(cd $DOC_REPOSITORY && find . -name "*.html" | xargs sed -i -e '\;<script type="application/vnd\.jupyter\.widget-state+json">;,\;</script>; d')
+# Create CHANGES.html
+echo '<html>' > CHANGES.html
+echo '<head>' >> CHANGES.html
+echo '<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/default.min.css">' >> CHANGES.html
+echo '<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/highlight.min.js"></script>' >> CHANGES.html
+echo '<script>hljs.highlightAll();</script>' >> CHANGES.html
+cat >> CHANGES.html << EOF
+<script>
+document.addEventListener('DOMContentLoaded', () => {
+const diffSite = 'https://pianomister.github.io/diffsite'
+const baseDocURL = 'https://sagemath-tobias.netlify.app'
+const diffParagraphs = document.querySelectorAll('p.diff');
+diffParagraphs.forEach(paragraph => {
+  const rootURL = window.location.origin;
+  const docAnchor = paragraph.querySelector('a');  // first "a" element
+  const url = new URL(docAnchor.href);
+  const path = url.pathname;
+  const anchor = document.createElement('a');
+  anchor.href = diffSite + '/?url1=' + rootURL + path + '&url2=' + baseDocURL + path;
+  anchor.textContent = 'compare with the base';
+  anchor.setAttribute('target', '_blank');
+  paragraph.appendChild(anchor);
+  paragraph.innerHTML += '&nbsp;';
+  const hunkAnchors = paragraph.querySelectorAll('a.hunk');
+  hunkAnchors.forEach(hunkAnchor => {
+    const url = new URL(hunkAnchor.href);
+    const path = url.pathname;
+    const pathHash = path + url.hash.replace('#', '%23');
+    const anchor = document.createElement('a');
+    anchor.href = diffSite + '/?url1=' + rootURL + pathHash + '&url2=' + baseDocURL + path;
+    anchor.textContent = hunkAnchor.textContent;
+    anchor.setAttribute('target', '_blank');
+    paragraph.appendChild(anchor);
+    paragraph.innerHTML += '&nbsp;';
+  });
+});
+});
+</script>
+EOF
+echo '</head>' >> CHANGES.html
+echo '<body>' >> CHANGES.html
+(cd $DOC_REPOSITORY && git diff $BASE_DOC_COMMIT -- *.html) > diff.txt
+/sage/sage -python - << EOF
```

Comment:
I think it would be better not to hardcode this path in the script

Reply:
Now the script is supplied the sage root (`/sage`)  as third argument. I don't have a better idea. Do you?